### ISSUE TITLE
[Views integration] How is caching supposed to work?

### ISSUE THREAD

**Issue by jensschuppe** [CiviMRF/cmrf_core#48]
https://github.com/CiviMRF/cmrf_core/blob/e3e3c9dfc4e4bb78c97580fd26abd22233df1db9/cmrf_views/src/Plugin/views/query/API.php#L225

This seems to have been ported from `7.x`, but I could neither get it to work nor do I understand where this caching option should be coming from.

Can anyone involved shed some light on whether Views caching (at least time-based) in the View's configuration is supposed to work with caching CiviMRF calls at all?

@fabioneves @jaapjansma @kainuk

**Comment by jaapjansma** [comment]
In Drupal 7 you can set a caching timeout in the views ui. This is per view.
This caching setting is then passed on to civi mc restface calls.

**Comment by jensschuppe** [comment]
Thanks, @jaapjansma, so this will need to be adapted for D8 since caching configuration in D8 is per Views display and can only be used if time-based caching is enabled, if I got that correctly.

Has anyone taken on that for D8 yet?

**Comment by jaapjansma** [comment]
Yes indeed this need to be adapted to d8. I also believe that the caching setting in d7 is per display of a view.

**Comment by jensschuppe** [comment]
Linking #62
